### PR TITLE
Finnish language file update

### PR DESCRIPTION
**Pull request by PotatoCreator**
The Finnish translations were incomplete, causing some user interface elements to fall back on the English default. This pull request aims to provide a fully localized experience for Finnish users.

Please note that builds and other checks will be run for your change. Don't feel intimidated by failures in some of the checks. If you can't resolve them yourself, experienced developers can also help resolve them before merging your pull request.

## Checklist

- [x] Completed missing translations for Finnish language file

### CODE REVIEW

File: data/languages/finnish.txt [hunk 1]
```diff
@@ -1059,783 +1061,771 @@ Best
 
 [Graphics error]
 Failed during initialization. Try to change gfx_backend to OpenGL or Vulkan in settings_ddnet.cfg in the config directory and try again.
-== 
+== [Grafiikkavirhe]
+Epäonnistui alustuksen aikana. Yritä muuttaa gfx_backend arvoksi OpenGL tai Vulkan asetushakemistossa settings_ddnet.cfg ja yritä uudelleen.
```
Comment (Robyt3) [inline data/languages/finnish.txt:1065]: The text in square brackets should not be translated, it only serves as additional context for the translator.

Add only the translated text on the line with the `==`. Also several times below.

**Review by Robyt3 — CHANGES_REQUESTED** [review]

**Comment by PotatoCreator** [comment]
Hope I didn't miss any.

**Comment by furo321** [comment]
You seemed to have accidentally deleted some strings from the file.

<details>
  <summary>Here are the ones that are currently missing.</summary>

```
Loading demo file from storage
== 

No login required
== 

Player info change cooldown
== 

Converse
== 

Always show chat
== 

Show friend mark (♥) in name plates
== 

Show hook strength icon indicator
== 

Show hook strength number indicator
== 

Show own player's hook collision line
== 

Always show own player's hook collision line
== 

Always show other players' hook collision lines
== 
```
</details>

File: data/languages/finnish.txt [hunk 2]
```diff
@@ -1,10 +1,12 @@
+
```
Comment (def-) [inline data/languages/finnish.txt:1]: This line was accidentally added?

**Review by def- — APPROVED** [review]

File: data/languages/finnish.txt (hunk 2 above)
Comment (PotatoCreator) [reply data/languages/finnish.txt:1]: I guess, can you edit it?

Comment (def-) [reply data/languages/finnish.txt:1]:
```suggestion

```

**Review by def- — APPROVED** [review]

**Pull request merged**